```
File "/b/f/w/bazel-out/k8-opt/bin/tensorflow/tools/docs/tf_doctest.runfiles/org_tensorflow/tensorflow/python/keras/activations.py", line 204, in tensorflow.python.keras.activations.relu
Failed example:
    tf.keras.activaions.relu(foo, max_value=5).numpy()
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python3.6/doctest.py", line 1330, in __run
        compileflags, 1), test.globs)
      File "<doctest tensorflow.python.keras.activations.relu[3]>", line 1, in <module>
        tf.keras.activaions.relu(foo, max_value=5).numpy()
    AttributeError: module 'tensorflow.python.keras.api._v2.keras' has no attribute 'activaions'
```

please fix

--- a/tensorflow/python/keras/activations.py
+++ b/tensorflow/python/keras/activations.py
@@ -201,7 +201,7 @@ def relu(x, alpha=0., max_value=None, threshold=0):
   array([ 0.,  0.,  0.,  5., 10.], dtype=float32)
   >>> tf.keras.activations.relu(foo, alpha=0.5).numpy()
   array([-5. , -2.5,  0. ,  5. , 10. ], dtype=float32)
-  >>> tf.keras.activaions.relu(foo, max_value=5).numpy()
+  >>> tf.keras.activations.relu(foo, max_value=5).numpy()
   array([0., 0., 0., 5., 5.], dtype=float32)
   >>> tf.keras.activations.relu(foo, threshold=5).numpy()
   array([-0., -0.,  0.,  0., 10.], dtype=float32)

--- a/tensorflow/python/keras/activations.py
+++ b/tensorflow/python/keras/activations.py
@@ -198,14 +198,13 @@ def relu(x, alpha=0., max_value=None, threshold=0):
   For example:
   >>> foo = tf.constant([-10, -5, 0.0, 5, 10], dtype = tf.float32)
   >>> tf.keras.activations.relu(foo).numpy()
-  # ==> array([0., 0., 0., 5., 10.], dtype=float32)
+  array([ 0.,  0.,  0.,  5., 10.], dtype=float32)
   >>> tf.keras.activations.relu(foo, alpha=0.5).numpy()
-  # ==> array([-5. , -2.5,  0. ,  5. , 10. ], dtype=float32)
+  array([-5. , -2.5,  0. ,  5. , 10. ], dtype=float32)
   >>> tf.keras.activaions.relu(foo, max_value=5).numpy()
-  # ==> array([0., 0., 0., 5., 5.], dtype=float32)
+  array([0., 0., 0., 5., 5.], dtype=float32)
   >>> tf.keras.activations.relu(foo, threshold=5).numpy()
   array([-0., -0.,  0.,  0., 10.], dtype=float32)
-
 
   Arguments:
       x: Input `tensor` or `variable`.

